azurerm/internal/services/compute/linux_virtual_machine_resource.go

```diff
@@ -177,6 +177,17 @@ func resourceLinuxVirtualMachine() *schema.Resource {
 
 			"identity": virtualMachineIdentitySchema(),
 
+			"license_type": {
+				Type:             schema.TypeString,
+				Optional:         true,
+				Computed:         true,
```

Comment:
Is there a reason for this to be `Computed`? I don't think this defaults to anything by the API so we shouldn't have any broken users unless they changed it outside of Terraform.

Reply:
It's computed in `virtual_machine` as well, so I made it the same. Want me to un-compute :) it in both?

Reply:
Hmmm, I see. The reason I'm calling it out is because if a user tries to empty the value, it won't actually empty it. I did some testing and it seems like you can't empty it anyway so this looks good!

Reply:
The legacy VM resource has a lot of outdated behaviour - in this instance this shouldn't be computed (and should be case-sensitive)